backends/qualcomm/scripts/build.sh

```diff
@@ -25,9 +25,9 @@ usage() {
 [ "$1" = -h ] && usage
 
 BUILD_X86_64="true"
-CMAKE_X86_64="cmake-out"
+CMAKE_X86_64="build-x86"
```

Comment:
@chiwwang I rename it to the similar name as before, because the ci job will clean up `cmake-out` for every run. We can make `build.sh` takes configurable name as next step and change the default name to `cmake-out` and `cmake-out-android`